tallycash/extension
# Add network-awareness to Base Assets

Closes https://github.com/tallycash/extension/issues/1614

This PR properly segments our BASE_ASSETs based on what network they belong to.

### To Test
- SUPPORT_POLYGON=true
- Import the `enroll` test wallet or `mhluongo.eth`. (or any wallet that definitely has a `0` balance of MATIC on mainnet
- Switch between polygon and ethereum as you please - MATIC should not show up in the wallet asset list when the selected ethereum is network
